### ISSUE TITLE
Help getting the word out.

### ISSUE THREAD

**Issue by martinheidegger** [dat-land/comm-comm#31]
In #29 we noticed that people are surprised about this meeting happening and we should announce it better.

My thoughts are immediately:

- Share through the [`@datproject`](https://twitter.com/dat_project/) twitter?
- Write an article about this on the datproject blog? @joehand 
- Share on SecureScuttleButt
- Do people use Facebook? ;)
- Create a calendar `.ics` file that is hosted on our homepage with all the calendar entries

**Comment by dpaez** [comment]
All of the above? :)

Thanks for articulating this into words, Martin!

**Comment by RangerMauve** [comment]
Posting in the IRC channel about an hour or two in advance might be good too.

**Comment by martinheidegger** [comment]
I think a calendar file should be the first thing, as it would open the doors to all sort of automation.

**Comment by joehand** [comment]
@dpaez is an author on the blog, @martinheidegger happy to send an invite to you as well if you'd like to collaborate on the post.

**Comment by martinheidegger** [comment]
@joehand sure!

**Comment by martinheidegger** [comment]
@RangerMauve #33 adds a calendar file, now just need some automation service ;)

**Comment by RangerMauve** [comment]
Just tried it out. It seems to be working. :D

**Comment by dpaez** [comment]
Hi all! just to let you know that I'm working on a new article about the dat comm-comm meetings, it's a draft yet but the idea is to talk about what is a comm-comm meeting, what can you do as a participant and goals. I'm going to put the article on ghost before publishing so others can suggest some ideas or fixes. 😊

**Comment by martinheidegger** [comment]
@dpaez I got a login to ghost but can not see other content (can you invite _?_ me to the draft article?)

**Comment by dpaez** [comment]
@martinheidegger yes my friend, I'll invite you as soon as the draft is available on ghost (because is not there yet, i need to fix/check some basic things first 😅)

**Comment by dpaez** [comment]
@joehand hi joe! I made a draft post and I would like to invite Martin to take a look at it, can you help us to be on the same team on ghost? It seems to have that feature but we can not see each other. 
Thanks!

**Comment by joehand** [comment]
Added you both as editors. Now you should be able to edit other posts and also invite other contributors =)

**Comment by dpaez** [comment]
Hi @joehand the [article](https://blog.datproject.org/ghost/#/editor/5c8ea7568fc5843ac5fbab9d) is ready, if you can give it a look it would be nice. If everything is ok we are ready to publish. 🆒

**Comment by joehand** [comment]
Looks great! Published it =)

**Comment by joehand** [comment]
Pinned tweet about it too: https://twitter.com/dat_project/status/1108816305447030784.

**Comment by martinheidegger** [comment]
Can someone help with automation around this? Send out twitter entry & irc message for events?
